internal/services/cdn/cdn_frontdoor_firewall_policy_resource_test.go

```diff
@@ -654,6 +752,17 @@ func TestAccCdnFrontDoorFirewallPolicy_jsChallengePolicyInvalidTimeSpanError(t *
 	})
 }
 
+func TestAccCdnFrontDoorFirewallPolicy_captchaPolicyInvalidTimeSpanError(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_cdn_frontdoor_firewall_policy", "test")
+	r := CdnFrontDoorFirewallPolicyResource{}
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config:      r.captchaPolicyInvalidTimeSpanError(data),
+			ExpectError: regexp.MustCompile(`expected captcha_cookie_expiration_in_minutes to be in the range \(5 - 1440\), got 4`),
```

Comment:
is this covered in the schema `ValidateFunc` ? Typically we don't include tests to check these, is there a reason why this being specifically tested here?

Reply:
Yes. It is covered by the schemas `ValidateFunc` but I included it in the tests to ensure that the core runtime catches the lower bound limit as expected.

Reply:
Fixed. I removed the test case and the js challenge test too which was checking essentially the same thing.